src/main/java/org/wildfly/security/auth/client/ElytronXmlParser.java

```diff
@@ -442,6 +447,12 @@ private static void parseSslContextType(final ConfigurationXMLStreamReader reade
                         credentialFactory = new PrivateKeyKeyStoreEntryCredentialFactory(parseKeyStoreRefType(reader, xmlVersion, keyStoresMap, credentialStoresMap, providersSupplier), location);
                         break;
                     }
+                    case "key-manager": {
```

Comment:
is it possible to implement "alternative" relation to key-store-ssl-certificate. So both of them could not be configured?